tests/components/rako/test_bridge.py

```diff
@@ -0,0 +1,65 @@
+"""Test the Rako Bridge logic."""
```

Comment:
Love the additional tests here, though: The design pattern followed is that tests should be in terms of the home assistant integration APIS (e.g. load integration, then assert which entities were loaded, their states, etc) rather than unit testing internals. The idea is to exercise the framerwork + integration and that it's all working right.

Reply:
Same with util